### PR TITLE
feat(docs): add Laravel install docs

### PR DESCRIPTION
**Pull request by rcrosbourne**

### CODE REVIEW

File: apps/www/content/docs/installation/laravel.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+### Configure components.json
+
+You will be asked a few questions to configure `components.json`:
+
+```txt showLineNumbers
+Would you like to use TypeScript (recommended)? no / yes
+Which style would you like to use? › Default
+Which color would you like to use as base color? › Slate
+Where is your global CSS file? › resources/app.css
```
Comment (dan5py) [inline apps/www/content/docs/installation/laravel.mdx:31]: The css file path should be:
```bash
resources/css/app.css
```

File: apps/www/content/docs/installation/laravel.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+Configure the import alias for components: › @/components
+Configure the import alias for utils: › @/lib/utils
+Are you using React Server Components? › no / yes
+```
+
+### Update tailwind.config.js
+
+The `shadcn-ui` CLI will automatically overwrite your `tailwind.config.js`. Update it to look like this:
+  ```js
+  import defaultTheme from "tailwindcss/defaultTheme";
```
Comment (dan5py) [inline apps/www/content/docs/installation/laravel.mdx:43]: Remove the indentation before `import defaultTheme`

File: apps/www/content/docs/installation/laravel.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+You can now start adding components to your project.
+
+```bash
+npx shadcn-ui@latest add button
+```
+
+The command above will add the `Button` component to your project. You can then import it like this:
+
+```tsx {1,6} showLineNumbers
+import { Button } from "@/components/ui/button"
```
Comment (dan5py) [inline apps/www/content/docs/installation/laravel.mdx:141]: The import path should be (keeping the project naming convention):
```bash
@/Components/ui/button
```

File: apps/www/content/docs/installation/laravel.mdx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+You will be asked a few questions to configure `components.json`:
+
+```txt showLineNumbers
+Would you like to use TypeScript (recommended)? no / yes
+Which style would you like to use? › Default
+Which color would you like to use as base color? › Slate
+Where is your global CSS file? › resources/app.css
+Do you want to use CSS variables for colors? › no / yes
+Where is your tailwind.config.js located? › tailwind.config.js
+Configure the import alias for components: › @/components
```
Comment (dan5py) [inline apps/www/content/docs/installation/laravel.mdx:34]: This should be:
```bash
@/Components
```

File: apps/www/content/docs/installation/laravel.mdx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+Configure the import alias for utils: › @/lib/utils
```
Comment (dan5py) [inline apps/www/content/docs/installation/laravel.mdx:35]: This should be:
```bash
@/Lib/utils
```

**Review by dan5py — CHANGES_REQUESTED** [review]
Thanks @rcrosbourne. Could you take a look at the changes I suggested?

File: apps/www/content/docs/installation/laravel.mdx (hunk 5 above)
Comment (rcrosbourne) [reply apps/www/content/docs/installation/laravel.mdx:35]: Thank you for the valuable feedback. I've made the changes suggested, but chose to retain the `lib` folder name as lowercase.

**Review by dan5py — APPROVED** [review]
👍 @shadcn

**Review by shadcn — APPROVED** [review]
Thank you!

**Pull request merged**